### DISCUSSION TITLE
[@types/react-calendar-timeline] Row Renderer Prop is not present in this package

### DISCUSSION THREAD

**Discussion by saivamsimorisetti** [DefinitelyTyped/DefinitelyTyped#64949] [Issues with a @types package]
Hello, We are using     [@types/react-calendar-timeline]: "^0.26.3" currently and rowRenderer prop is not defined in this particular package due to which we are unable to use rowRenderer prop in the Timeline component of react calendar timeline

[a comment by an automated account is not shown]

**Comment by RIP21** [comment]
I haven't used this module for at least 4 years or so. So no input from me here :)
